S/seatd/bundled/patches/fix_crosscompile.patch

```diff
@@ -1,33 +1,20 @@
 --- meson.build
-+++ meson.build
-@@ -33,10 +33,10 @@
- 		'-Wold-style-definition', # nop
- 		'-Wpointer-arith',
- 		'-Wstrict-prototypes',
--		'-Wimplicit-fallthrough',
-+#		'-Wimplicit-fallthrough',
- 		'-Wmissing-prototypes',
--		'-Wno-unknown-warning-option',
--		'-Wno-unused-command-line-argument',
-+#		'-Wno-unknown-warning-option',
-+#		'-Wno-unused-command-line-argument',
- 		'-Wvla',
- 		'-Wl,--exclude-libs=ALL',
- 		'-D_XOPEN_SOURCE=700',
-@@ -153,7 +153,7 @@
- 	link_with: private_lib,
- 	include_directories: [include_directories('.', 'include')],
- 	install: true,
--	link_args: symbols_flag,
-+	link_args: ['-lrt', symbols_flag],
- 	link_depends: symbols_file,
- )
++++ meson.build2
+@@ -132,6 +132,10 @@
+ 	private_deps += logind
+ endif
  
-@@ -173,6 +173,7 @@
++# needed for cross-compilation
```

Comment:
For what is worth, I'm not sure the comment is accurate

Reply:
Is it just because of other things about BB's setup?

Reply:
`librt` is _always_ needed, because `clock_gettime` lives there.  Specifying the library as dependency is the right thing to do regardless.  However in some cases it's pulled in automatically, here we have to link to it explicitly.

Reply:
Yeah, I agree it's not 100% accurate. Is it worth changing? It has been merged upstream already.

Reply:
Well, at this point I guess it doesn't really matter.